ocs_ci/ocs/must_gather/must_gather.py

```
@@ -101,14 +101,36 @@ def search_file_path(self):
-                    break
+            if file in basenames:
+                self.files_path[file] = basenames[file]
+            elif (
+                ec_pool
+                and DEFAULT_CEPHBLOCKPOOL in file
+                and file.replace(DEFAULT_CEPHBLOCKPOOL, ec_pool) in basenames
+            ):
+                ec_file = file.replace(DEFAULT_CEPHBLOCKPOOL, ec_pool)
+                logger.info(
+                    f"Must-gather file '{file}' not found, matched EC "
+                    f"variant '{ec_file}'"
```

Ruff (0.15.20)

[warning] Logging statement uses f-string

(G004)